ts/features/bonusVacanze/screens/ActivateBonus/ActivateBonusScreen.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { RTron } from "../../../../boot/configureStoreAndPersistor";
+import { shufflePinPadOnPayment } from "../../../../config";
+import I18n from "../../../../i18n";
+import { identificationRequest } from "../../../../store/actions/identification";
+import { GlobalState } from "../../../../store/reducers/types";
+import { mockedBonus, mockedIseeFamilyMembers } from "../../mock/mockData";
+import { ActivateBonusComponent } from "./ActivateBonusComponent";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+/**
+ * This screen allows the user to visualize the amount of the bonus available, the family composition, and offers two CTA:
+ * - Cancel: Abort the bonus request
+ * - Activate the bonus: Start the procedure to activate the bonus
+ * The screen is tied to the business logic and is composed using {@link ActivateBonusComponent}
+ * @param props
+ * @constructor
+ */
+const ActivateBonusScreen: React.FunctionComponent<Props> = props => {
+  return (
+    <ActivateBonusComponent
+      onCancel={props.onCancel}
+      onRequestBonus={props.onActivateBonus}
+      bonusAmount={props.bonusVacanze.max_amount}
+      familyMembers={props.familyMembers}
+    />
+  );
+};
+
+/**
+ * Verify the identification using pin / biometric
+ * @param dispatch
+ */
+const requestIdentification = (dispatch: Dispatch) => {
+  dispatch(
+    identificationRequest(
+      false,
+      true,
+      {
+        message: I18n.t("bonus.bonusVacanza.activateBonus.title")
+      },
+      {
+        label: I18n.t("global.buttons.cancel"),
+        onCancel: () => undefined
+      },
+      {
+        onSuccess: onIdentificationSuccess
+      },
+      shufflePinPadOnPayment
+    )
+  );
+};
+
+const onIdentificationSuccess = () => {
+  // TODO: link with the business logic that dispatch the activation to backend
+  RTron.log("IdentificationSuccess");
+};
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  // TODO: link with the right dispatch action
+  onCancel: () => {
+    RTron.log("CANCEL");
+  },
+  // When the user choose to activate the bonus, verify the identification
+  onActivateBonus: () => requestIdentification(dispatch)
+});
+
+const mapStateToProps = (_: GlobalState) => ({
+  // TODO: link with the right reducer
+  bonusVacanze: mockedBonus,
```

Comment:
@Undermaken Are we sure to use this data model here? In this phase we should have only the response from `/eligibility` with the payload with values `family_members`, `max_amount` and `max_tax_benefit`. Seems to me that the model `BonusVacanzaMock` represents an already request bonus.

Reply:
@fabriziofff Yes, you are totally right.
I lost the point where we are.
At this point we should have something like this

```json
{
  "family_members": [
    {
      "name": "Mario",
      "surname": "Rossi",
      "fiscal_code": "EFCMZZ80A12L720R"
    },
    {
      "name": "Giulia",
      "surname": "Rossi",
      "fiscal_code": "CDCMQQ81A12L721R"
    },
    {
      "name": "Piero",
      "surname": "Rossi",
      "fiscal_code": "ABCMYY82A12L722R"
    }
  ],
  "max_amount": 50000,
  "max_tax_benefit": 3000,
  "id": "76d02f6d-ca02-4b1c-8b6a-4b8ca71d6293",
  "status": "ELIGIBLE"
}
```
Please could you revert?